src/lms-harmonizer/tests_integration_sql/procs/mssql/harmonize_assignments/test_lmsassignments.py

```diff
@@ -98,11 +98,49 @@ def it_should_run_successfully(test_db_config: ServerConfig, source_system: str)
 
         # assert
         with MSSqlConnection(test_db_config).pyodbc_conn() as connection:
-            LMSAssignment = query(connection, "SELECT * from [lmsx].[Assignment]")
-            assert len(LMSAssignment) == 1
-            assert (
-                int(LMSAssignment[0]["AssignmentIdentifier"]) == 1
-            )  # It is using the identity field from lms.Assignment
+            result = query(connection, "SELECT * from [lmsx].[Assignment]")
+
+        assert len(result) == 1, "There should be one result."
+
+        LMSAssignment = result[0]
+
+        # TODO: consider looking up the correct expected value in the Descriptor table.
+        # For nowjust validate that _something_ was set
+        assert (
+            int(LMSAssignment["AssignmentIdentifier"]) == 1
+        ), "It should map the assignment identifier"
+
+        assert (
+            LMSAssignment["LMSSourceSystemDescriptorId"] is not None
+        ), "It should map the SourceSystem descriptor"
+
+        assert (
+            LMSAssignment["AssignmentCategoryDescriptorId"] is not None
+        ), "It should map the assignment category descriptor"
+
+        assert (
+            LMSAssignment["SectionIdentifier"] is not None
+        ), "It should map the section identifier"
+
+        assert (
+            LMSAssignment["LocalCourseCode"] is not None
+        ), "It should map the local course code"
+
+        assert (
+            LMSAssignment["SessionName"] is not None
+        ), "It should map the SessionName"
+
+        assert (
+            LMSAssignment["SchoolYear"] is not None
+        ), "It should map the SchoolYear"
+
+        assert (
+            LMSAssignment["SchoolId"] is not None
```

Comment:
The `SchoolId` value is 1. See line 291 in `mssql_loader.py`. Other values can be known based on constants. So we should only do the  "is not None" test for those values that are hard for us to know - that aren't hard-coded somewhere in our Python modules..

Reply:
So please re-arrange so that the values that _can't_ easily be known - descriptor IDs are really the only ones - are at the end of the list. All of the other asserts should be above the comment, and should look for the specific known values.